autoswitch_virtualenv.plugin.zsh

```diff
@@ -17,6 +17,22 @@ function maybeworkon() {
   fi
 }
 
+# Gives the path to the nearest parent .venv file or nothing if it gets to root
+function check_venv_path()
+{
+    local check_dir=$1
+
+    if [[ -f "${check_dir}/.venv" ]]; then
+        echo "${check_dir}/.venv"
+        return
```

Comment:
Does zsh support returning strings? My understanding was that return values were only for exit status.